django/views/debug.py

```diff
@@ -24,7 +24,7 @@
     libraries={'i18n': 'django.templatetags.i18n'},
 )
 
-HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE', flags=re.IGNORECASE)
+HIDDEN_SETTINGS = re.compile('API|TOKEN|KEY|SECRET|PASS|SIGNATURE|URL|DSN', flags=re.IGNORECASE)
```

Comment:
I wonder, maybe in future we should introduce a `django.conf` setting for this regexp

Reply:
Maybe, on the other hand forcing people to use a specific (relatively) small set of names for sensitive data might be actually not so bad.

Reply:
Also, https://github.com/django/django/blob/master/docs/ref/settings.txt#L1134 needs updating, it seems

Reply:
@apollo13 with that, I agree. Providing _some_ way to add more hidden items without too much of hack would be nice though.